py/picca/fitter2/chi2.py

```diff
@@ -228,6 +228,10 @@ def export(self):
         for (p1, p2), cov in self.best_fit.covariance.items():
             g.attrs["cov[{}, {}]".format(p1,p2)] = cov
 
+        if len(priors.prior_dic) != 0:
+            for prior in priors.prior_dic.values():
+                g.attrs["prior[{}]".format(prior.keywords['name'])] = (prior.keywords['prior_pars'][0],prior.keywords['prior_pars'][1],prior.func.__name__.encode('utf8'))
```

Comment:
same as my comment, `prior.keywords['prior_pars'][0]` is hard to generalize for other type of priors.